OrbitCore/FramepointerValidatorService.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "FramepointerValidatorService.h"
+
+#include <capstone/capstone.h>
+#include <vector>
+
+#include "FunctionFramepointerValidator.h"
+#include "OrbitFunction.h"
+#include "OrbitModule.h"
+#include "OrbitProcess.h"
+#include "Pdb.h"
+
+FramepointerValidatorService::FramepointerValidatorService(
+    const ProcessList* process_list, TransactionService* transaction_service)
+    : process_list_{process_list}, transaction_service_{transaction_service} {
+  auto on_request = [this](const Message& msg) { HandleRequest(msg); };
+  transaction_service_->RegisterTransactionRequestHandler(
+      {on_request, Msg_ValidateFramepointer, "Validate Framepointers"});
+}
+
+void FramepointerValidatorService::HandleRequest(const Message& message) {
+  // TODO: The code below is about 90% copy&past from SymbolsService.cpp.
+  //  We should discuss.
+  std::vector<ModuleDebugInfo> module_infos;
+  transaction_service_->ReceiveRequest(message, &module_infos);
+
+  for (const auto& module_info : module_infos) {
+    // Find process.
+    uint32_t pid = module_info.m_PID;
+    std::shared_ptr<Process> process = nullptr;
+    process = process_list_->GetProcess(pid);
+    if (process == nullptr) {
+      ERROR("Unable to find process %u", pid);
+      continue;
+    }
+
+    const bool is64Bit = process->GetIs64Bit();
+
+    // Find module.
+    const std::string& module_name = module_info.m_Name;
+    std::shared_ptr<Module> module = process->GetModuleFromName(module_name);
+    if (!module) {
+      ERROR("Unable to find module %s", module_name.c_str());
```

Comment:
Maybe delimit the module's name. E.g.:
`ERROR("Unable to find module \"%s\"", module_name.c_str());`
or
`ERROR("Unable to find module: %s", module_name.c_str());`

Also you don't need `c_str()` as `absl::StrFormat` is used behind the scenes.